need compatibility with cross-env package

Thank you for taking time to open a new issue. Please answer a few questions to help us fix it faster. You can delete text that is irrelevant to the issue.

## Is this a bug report or a feature request?
Bug
If this is a bug report, please provide as much info as possible

- version
1.5
- platform
mac
- expected behavior
no error after exist
- actual behavior
error after test finishes and exits.


-Detail:
I am using Jest library as testing package.
If I use cross-env to define environment variables for both test and start-server, then it will throw errors when the Jest test exists.
My current configuration:
    "test": "cross-env NODE_ENV=test  jest --forceExist",
    "test-server": "cross-env NODE_ENV=test  ts-node src/index.ts",
    "ci": "start-server-and-test test-server http://localhost:4000 test",

the error message when it exists the test is the following:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! typeorm@0.0.1 test-server: `cross-env NODE_ENV=test  ts-node src/index.ts`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the typeorm@0.0.1 test-server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tanekim/.npm/_logs/2018-07-05T23_57_30_700Z-debug.log


If I remove the keywords cross-env from all lines and then run ci again, then I don't see the error message.
Also, it is a problem with mac, not windows, as I tested.



If this is a new feature request, please describe it below

--- a/src/index.js
+++ b/src/index.js
@@ -36,7 +36,7 @@ function startAndTest ({ start, url, test }) {
           debug('stopping child processes')
           children.forEach(child => {
             try {
-              process.kill(child.PID)
+              process.kill(child.PID, 'SIGINT')
             } catch (e) {
               if (e.code === 'ESRCH') {
                 console.log(`Child process ${child.PID} exited before trying to stop it`)

